.github/workflows/CI.yml

```diff
@@ -295,6 +295,7 @@ jobs:
       name: "Windows: Package gtk"
       shell: bash
       run: |
+        export PATH="/usr/${{ steps.vars.outputs.MinGW_ARCH }}-w64-mingw32/bin":${PATH}
```

Comment:
If I'm following, this is a workaround for a bug (which is not clearly understood) in GHA-land, in that PATH should have that included already, but has gotten broken.   This seems like a fine fix, but I'd like to see a comment in the code (rather than just the commit message) to explain (to future me, plus others) why a setting that shouldn't have to exist is present.